test/unit/argument_parser/format_parse_validators_test.cpp

```diff
@@ -52,6 +53,21 @@ TEST(validator_test, fullfill_concept)
     EXPECT_TRUE(validator_concept<file_ext_validator>);
 }
 
+TEST(validator_test, no_file)
+{
+  filesystem::path p{"./sandbox.fasta"};
```

Comment:
4 spaces instead of 2 :nerd_face: 
Usually you can set up you editor to automatically indent by 4 when using tab.

Could you also add a string variable for each test:
```cpp
    filesystem::path p{"./sandbox.fasta"};
    std:;string s{"./stonebox.fasta"};
    file_existance_validator my_validator{};
    EXPECT_THROW(my_validator(p), parser_invalid_argument);
    EXPECT_THROW(my_validator(s), parser_invalid_argument);
``` 
because it should work on strings